region/norfair/west/Crocomire Escape.json

```diff
@@ -844,6 +844,22 @@
         "Also requires a crouchjump and downgrab to complete the maneuver."
       ]
     },
+    {
+      "link": [2, 3],
+      "name": "Spring Ball Jump into Damage Boost",
+      "requires": [
+        {"notable": "Spring Ball Jump into Damage Boost"},
+        {"heatFrames": 1600},
```

Comment:
You need to add RNG protection.  2 more springball jumps makes it better, but not guaranteed still, at 32 seconds=1920frames.

Reply:
Makes sense. I didn't practice it enough to understand the patterns, just assumed I had messed it up when it didn't work sometimes. Increased the heat frames to 1920 now.